packages/scripts/src/split-file/write-files.js

```diff
@@ -0,0 +1,91 @@
+import { existsSync, mkdirSync, writeFileSync } from 'fs';
+import path from 'path';
+import { camelToKebab } from './utils.js';
+
+/**
+ * @param {string} str
+ * @returns {string}
+ */
+const extendPaths = (str) =>
```

Comment:
adds an extra `../` to paths so they point in the right direction

Reply:
Actually I realise now this is dependent on the output dir being one nest deep in the original files dir. I'll clamp down on output dir specification so this always works